cves/kernel/CVE-2017-10911.yml

```diff
@@ -75,7 +78,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1458870] 
```

Comment:
This bug ID is not returning a result either? Are you getting these IDs from [bugzilla.kernel.org](bugzilla.kernel.org)?